### ISSUE TITLE
Voxelization  in transformer

### ISSUE THREAD

**Issue by r03943158** [facebookresearch/DepthContrast#31]
Hi, thank you provide the code of the amazing works!

I am curious about the voxel-level input. In the process of voxelization, the feature of the voxel adopts the feature of the point that has a minimum index instead of using the average of all the points in the voxel grid.  Is this related to MinkowskiEngine?

Thank you!

**Comment by zaiweizhang** [comment]
Yeah. That's related to he MinkowskiEngine. I borrowed the voxelization process code from the corresponding downstream tasks.
